jobs/candlepinPerformanceTestsJob.groovy

```diff
@@ -20,7 +20,6 @@ job("$baseFolder/CandlepinPerformance") {
         stringParam('candlepin_throughput_properties', '[\'DURATION_SECONDS=3600\',\'SAMPLES_PER_MINUTE=6900\']', 'override test duration, sample size(small=180, medium = 900, large=6900), and other properties.')
         stringParam('loop_over_apis_properties', '[\'SCALE=1000\']', 'override test scale or other specific loop counts.')
         choiceParam('jmeter_tests', ['candlepin-throughput','loop-over-apis','ImportExport'], 'what test to run')
-        choiceParam('target_branch', ['master', 'candlepin-2.0-HOTFIX','candlepin-2.1-HOTFIX'], 'what is target branch of PR (influences snapshot of database VM use for testing)')
```

Comment:
need to remove  the usage later in the file as well @jirihnidek

Reply:
@vritant good point. The `target_branch` is also removed from `--extra-vars` now.